src/components/LibrariesListItem.tsx

```diff
@@ -12,43 +12,55 @@ export interface LibrariesListItemProps {
 
 export interface LibrariesListItemState {
   color: string;
+  panelOpen: boolean;
 }
 
 /** A color-coded panel displayed for each item in LibraryList, containing a LibraryDetailPage. */
-export default class LibrariesListItem extends React.Component<LibrariesListItemProps, LibrariesListItemState> {
+export default class LibrariesListItem extends React.Component<
+  LibrariesListItemProps,
+  LibrariesListItemState
+> {
   constructor(props: LibrariesListItemProps) {
     super(props);
     const color = this.colorCode(Object.values(this.props.library.stages));
-    this.state = { color };
+    this.state = { color, panelOpen: false };
```

Comment:
You may want to set this to `this.props.panelOpen` instead of `false`, right? If you pass in `this.props.panelOpen=true` on the initial render, it won't be opened as expected in the initial render.

Reply:
Same as before, this is the state and not the prop so this is okay to leave as is. The syntax confused me.